assets/src/domain/index.ts

```diff
@@ -3,15 +3,12 @@ import './shared/services/publicPath';
 const domain = window?.eeDomain;
 
 if (domain) {
+	console.log(`%c importing: /domain/${domain}/entryPoint.ts`, 'color: SkyBlue;');
 	import(
 		/* webpackExclude: /(shared)/ */
 		/* webpackChunkName: "[request]" */
 		`./${domain}/entryPoint.ts`
-	)
-		.then(() => {
-			console.log('domain', domain);
-		})
-		.catch(console.error);
```

Comment:
Vadim won't be happy with this 😄

Reply:
just checked and found this answer on stackoverflow of this exact situation and it's possible to use `.catch` w/o `.then`: https://stackoverflow.com/a/36213268
***
not a biggie but the `console.log` from the beginning will be printed even if we'd have errors, but as it was before it was printing only after successful import

Reply:
ya I changed it because when trying to work on this issue, I wasn't getting anything printed at first, because it wasn't resolving successfully but there also wasn't an error

I'm fine with this being changed back to something else, but I DO agree with that SO comment that a `.then()` clause is not needed